### PR TITLE
izle7.com #93757

### LINKED ISSUE #93757: www.izle7.com
### Issue URL (Ads)
 [https://www.izle7.com/kanal7/izle-53394-dirilis-ertugrul-55-bolum-izle.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.izle7.com%2Fkanal7%2Fizle-53394-dirilis-ertugrul-55-bolum-izle.html)
### Comment
Username: @RadicalNoise
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.6.12
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by ghost**
Fix #93757 

Also obsolete rules removed
d80f895
c0e2d1c

### CODE REVIEW

**Comment by RadicalNoise** [comment]
`onlinedizi.tv` update is not related to this PR. 
PR for `onlinedizi.tv` has already been closed. 67a5c57

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -1880,9 +1882,8 @@ e-inegol.com###modal-ads ~ div.modal-backdrop
 dizitime.pw##.col.pr-0 a[href][target="_blank"] > img
 ||dizitime.pw/images/dtm-promo.gif
 ||dizitime.pw/images/dtm-deposit.gif
-onlinedizi.net##.headerbnr
-onlinedizi.net##.banner-container
-||onlinedizi.net/images/header*.js
+onlinedizi.tv##.banner-container
+||onlinedizi.tv/images/header*.js
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:1886]: I don't see this request.

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -2308,10 +2309,10 @@ haberler.com###DetailMtrxCentre > div#quiz_masthead[style^="width:970px; height:
 ||haber50.com/*/images/banner/
 freemagazine.com.tr##.u-ads
 freemagazine.com.tr##.sagads
-onlinedizi.net##.video_top_banner
-onlinedizi.net##body > a[href][target="_blank"][rel="nofollow"]
-||onlinedizi.net/images/video_top.js
-||onlinedizi.net/images/betwinner-bg.jpg
+onlinedizi.tv##.video_top_banner
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:2312]: No elements and requests - obsolete rules:

onlinedizi.tv##.video_top_banner
onlinedizi.tv##body > a[href][target="_blank"][rel="nofollow"]
||onlinedizi.tv/images/video_top.js
||onlinedizi.tv/images/betwinner-bg.jpg

File: TurkishFilter/sections/specific.txt [hunk 3]
```diff
@@ -2577,7 +2578,7 @@ dolufilmizle.com,fullhdiziizle.com##.wrapper.rkads
 /wp-content/uploads/*/*/*reklam*.*$image,domain=buyuksivas.com
 aktuelpsikoloji.com##.banner > a[target^="_blan"] > img
 turizmajansi.com##a[href^="/rdads.aspx?r="]
-diziboxx.com,dizimag.cc,onlinedizi.net###pre_video
+diziboxx.com,dizimag.cc,onlinedizi.tv###pre_video
```
Comment (Alex-302) [inline TurkishFilter/sections/specific.txt:2581]: obsolete

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Comment by RadicalNoise** [comment]
@Alex-302 

Just focus on the rules about `izle7.com` If the rules are correct please add manually and close this issue.

`onlinedizi.tv` is a different pull request and has already been fixed before. See #93922

There is a confusion here, and the confusion with this pull request is my fault. Sorry about that.

**Comment by zloyden** [comment]
Fixed directly.

**Pull request closed without merging**